# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1380758144 14400
#      Wed Oct 02 19:55:44 2013 -0400
# Node ID 2eeea8b67b63cfdc1eea4910dbf076f7f1279fbd
# Parent  fc31cd1bd503a210c03ba47d1dab580049b91ece
fix incorrect function name

* @ftp/saveobj.m: Call __ftp_pwd__, not __ftp_pwd.

diff --git a/scripts/@ftp/saveobj.m b/scripts/@ftp/saveobj.m
--- a/scripts/@ftp/saveobj.m
+++ b/scripts/@ftp/saveobj.m
@@ -14,11 +14,11 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 function b = saveobj (a)
   b = a;
   b = rmfield (b, "curlhandle");
-  b.dir = __ftp_pwd (a.curlhandle);
+  b.dir = __ftp_pwd__ (a.curlhandle);
 endfunction
 
